### DISCUSSION TITLE
how to have moderators when using discord integration

### DISCUSSION THREAD

**Discussion by djay** [mozilla/hubs#3711] [Hubs Cloud Questions] [answered]
I think its a bug but I'm not sure. no matter what I do I can't seem to elevate certain users permissions within a room based on permissions in discord - https://github.com/mozilla/hubs/issues/3710

Am I doing something wrong or is it a bug or is it just something not working with the public hubs rooms?

**Comment by robinkwilson** [comment, marked as answer]
Thank you @djay for the PR to the docs update! 

Because of djay's docs PR [here](https://github.com/MozillaReality/hubs-docs/pull/91/files), we now have shiny discord bot permissions section to answer the above question on this page: [Hubs Docs: Discord Bot page](https://hubs.mozilla.com/docs/hubs-discord-bot.html).
